polymesh_schema.json

```diff
@@ -105,6 +105,11 @@
             "primary_key": "AccountId",
             "secondary_keys": "Vec<SecondaryKey>"
         },
+        "KeyIdData": {
```

Comment:
@Centril you also need to add the RPC function structure to the schema that allows SDK to know about the existence of the RPC. 

EX- 

```
"portfolio": {
            "getPortfolios": {
                "description": "Gets all user-defined portfolio names of an identity",
                "params": [
                    {
                        "name": "did",
                        "type": "IdentityId",
                        "isOptional": false
                    }
                ],
                "type": "GetPortfoliosResult"
            },
```

Reply:
Done